config/201-clusterrole.yaml

```diff
@@ -110,6 +110,7 @@ rules:
   resources:
     - configmaps
     - pods # For reading termination messages
+    - secrets
```

Comment:
It seems that we can be even more specific here, and say only be able to list google-cloud-key... See here:
https://kubernetes.io/docs/concepts/configuration/secret/#clients-that-use-the-secret-api
So I think it will have to move secrets to a separate block, and add resourceNames to google-cloud-key... https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources... 

Not a strong opinion, but if we see it works, I think it will be better to narrow down the permissions